compat/src/Children.js

```diff
@@ -1,7 +1,7 @@
 import { toChildArray } from 'preact';
 
 const mapFn = (children, fn) => {
-	if (!children) return null;
+	if (typeof children === 'undefined' || children === null) return null;
```

Comment:
```suggestion
	if (children === undefined || children === null) return null;
```

Reply:
I think this check more robust, since we can overwrite `undefined`variable.